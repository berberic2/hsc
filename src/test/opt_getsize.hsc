<HTML><HEAD><TITLE>Test Image sizes</TITLE></HEAD><BODY>

<IMG SRC="test.gif" ALT="test">
<IMG SRC="test.png" ALT="test">
<IMG SRC="test.jfif" ALT="test">

<IMG SRC="test.ilbm" ALT="test"> <* will be ignored *>

</BODY></HTML>
