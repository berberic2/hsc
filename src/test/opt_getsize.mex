test/opt_getsize.hsc (7,33): Warning 68: filetype of `test/test.ilbm' not recognised
