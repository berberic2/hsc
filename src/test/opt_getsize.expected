<HTML><HEAD><TITLE>Test Image sizes</TITLE></HEAD><BODY>

<IMG SRC="test.gif" ALT="test" WIDTH="320" HEIGHT="256">
<IMG SRC="test.png" ALT="test" WIDTH="320" HEIGHT="256">
<IMG SRC="test.jfif" ALT="test" WIDTH="320" HEIGHT="256">

<IMG SRC="test.ilbm" ALT="test"> 
</BODY></HTML>
